conda/recipes/libcugraph/recipe.yaml

```diff
@@ -0,0 +1,268 @@
+        SCCACHE_S3_KEY_PREFIX: libcugraph/${{ env.get("RAPIDS_CONDA_ARCH") }}/cuda${{ cuda_major }}
+
+  requirements:
+    build:
+      - ${{ compiler("c") }}
+      - ${{ compiler("cxx") }}
+      - ${{ compiler("cuda") }} =${{ cuda_version }}
+      - cuda-version =${{ cuda_version }}
+      - cmake ${{ cmake_version }}
+      - ninja
+      - ${{ stdlib("c") }}
+    host:
+      - cuda-version =${{ cuda_version }}
+      - libcudf =${{ minor_version }}
+      - libraft =${{ minor_version }}
+      - librmm =${{ minor_version }}
+      - libucxx ${{ libucxx_version }}
+      - nccl ${{ nccl_version }}
+      - openmpi <5.0.3 # Required for building cpp-mgtests (multi-GPU tests)
+      - rapids-build-backend >=0.3.1,<0.4.0.dev0
+      - if: cuda_major == "11"
+        then:
+          - cudatoolkit
+          - cuda-nvtx =${{ cuda_version }}
+          - cuda-profiler-api ${{ cuda11_cuda_profiler_api_host_version }}
+          - libcublas ${{ cuda11_libcublas_host_version }}
+          - libcublas-dev ${{ cuda11_libcublas_host_version }}
+          - libcurand ${{ cuda11_libcurand_host_version }}
+          - libcurand-dev ${{ cuda11_libcurand_host_version }}
+          - libcusolver ${{ cuda11_libcusolver_host_version }}
+          - libcusolver-dev ${{ cuda11_libcusolver_host_version }}
+          - libcusparse ${{ cuda11_libcusparse_host_version }}
+          - libcusparse-dev ${{ cuda11_libcusparse_host_version }}
+        else:
+          - cuda-nvtx-dev
+          - cuda-profiler-api
+          - cuda-cudart-dev
+          - libcublas-dev
+          - libcurand-dev
+          - libcusolver-dev
+          - libcusparse-dev
+
+outputs:
+  - package:
+      name: libcugraph
+      version: ${{ version }}
+    build:
+      script:
+        content: |
+          cmake --install cpp/build
+      dynamic_linking:
+        overlinking_behavior: "error"
+      prefix_detection:
+        ignore:
+          # See https://github.com/rapidsai/build-planning/issues/160
+          - lib/libcugraph.so
+          - lib/libcugraph_c.so
+      string: cuda${{ cuda_major }}_${{ date_string }}_${{ head_rev }}
+    requirements:
+      build:
+        - cmake ${{ cmake_version }}
+        - ${{ stdlib("c") }}
+      host:
+        - cuda-version =${{ cuda_version }}
+        - rapids-logger =0.1
+        - if: cuda_major == "11"
+          then:
+            - cudatoolkit
+            - libcublas-dev ${{ cuda11_libcublas_host_version }}
+            - libcurand-dev ${{ cuda11_libcurand_host_version }}
+            - libcusolver-dev ${{ cuda11_libcusolver_host_version }}
+            - libcusparse-dev ${{ cuda11_libcusparse_host_version }}
+          else:
+            - cuda-cudart-dev
+            - libcublas-dev
+            - libcurand-dev
+            - libcusolver-dev
+            - libcusparse-dev
+      run:
+        - ${{ pin_compatible("cuda-version", upper_bound="x", lower_bound="x") }}
+        - libraft =${{ minor_version }}
```

Comment:
A couple checks on the RAPIDS dependencies:

Do we need a `libraft` or `librmm` in `host` too? Or just in the top-level cache?

And are we relying on `rapids-logger`'s run-exports to put it into `run`?

Reply:
There were no linking errors, so I think we're good?  

`rapids-logger` comes with `librmm` (which is also how `cugraph` loads it, so that seems consistent?)

Reply:
Currently we need `librmm` in `run` for every package using it at build time, to act as an ABI constraint. This will become a real runtime constraint with https://github.com/rapidsai/rmm/issues/1779 when RMM becomes a shared library.